lib/MusicBrainz/Server/WebService/XMLSerializer.pm

```diff
@@ -511,6 +511,7 @@ sub _serialize_work
             $gen->attribute({ 
                 type => $_->type->name,
                 "type-id" => $_->type->gid,
+                "value-id" => $_->value_gid,
```

Comment:
Will that output something if `value_gid` is undef? If so you might need to do something like `$_->type->free_text ? () : ("value-id" => $_->value_gid),`.